### PR TITLE
fix(migrate reset): exit 1 if an error happens during seeding

### PR DESCRIPTION
**Pull request by Jolg42**
Closes https://github.com/prisma/prisma/issues/9786

### CODE REVIEW

File: packages/migrate/src/__tests__/MigrateReset.test.ts [hunk 1]
```diff
@@ -239,6 +239,36 @@ describe('reset', () => {
     expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
   })
 
+  testIf(process.platform !== 'win32')('reset - seed.js - error should exit 1', async () => {
+    const mockExit = jest.spyOn(process, 'exit').mockImplementation()
+    ctx.fixture('seed-sqlite-js')
+    ctx.fs.write('prisma/seed.js', 'BROKEN_CODE_SHOULD_ERROR;')
+    prompt.inject(['y']) // simulate user yes input
```
Comment (Jolg42) [inline packages/migrate/src/__tests__/MigrateReset.test.ts:246]: Reproduction here, @jkomyno 
```
cd packages/migrate/src/__tests__/fixtures/seed-sqlite-js
echo "BROKEN_CODE_SHOULD_ERROR" > prisma/seed.js
# Should exit 0
npx prisma migrate reset && echo "exit 0" || echo "exit 1"
# with the change, should exit 1
../../../bin.ts migrate reset && echo "exit 0" || echo "exit 1"
```

Comment (jkomyno) [reply packages/migrate/src/__tests__/MigrateReset.test.ts:246]: I confirm the reproduction works.

**Review by jkomyno — APPROVED** [review]
We can ship this for me, but I have added a few TODO comments to address in future PRs

File: packages/migrate/src/commands/MigrateDev.ts [hunk 2]
```diff
@@ -325,8 +325,11 @@ ${chalk.green('Your database is now in sync with your schema.')}`,
           console.info() // empty line
           const successfulSeeding = await executeSeedCommand(seedCommandFromPkgJson)
           if (successfulSeeding) {
+            // TODO: what about Windows users running prisma on Git Bash or similar? Technically, they'd
+            // still be able to support emojis in their terminal.
```
Comment (janpio) [inline packages/migrate/src/commands/MigrateDev.ts:329]: That is not a TODO in code, but a `kind/tech` issue to be created.

Comment (jkomyno) [reply packages/migrate/src/commands/MigrateDev.ts:329]: Added https://github.com/prisma/prisma/issues/13355, I will remove the TODO in the code

**Pull request merged**